src/bluesearch/embedding_models.py

```diff
@@ -492,16 +321,17 @@ def get_embedding_model(model_name_or_class, checkpoint_path=None, device=None):
 
     Parameters
     ----------
-    model_name_or_class : str
+    model_name_or_class :
```

Comment:
So I guess you removed the type information from the docstring on purpose since we have annotations, right?
In that case should we maybe remove the `:`? I am not sure what the convention is.

Reply:
Fixed in 1f6eec6. By the way, I noticed that it is necessary to specify the return type of the function in the docstring or Sphinx will get confused. 

For instance, if we set the docstring
```python
   """
    Returns
    -------
    sentence_embedding_model : EmbeddingModel
        The sentence embedding model instance.
   """
```
we correctly get 
<img width="529" alt="Screenshot 2021-03-17 at 14 51 15" src="https://user-images.githubusercontent.com/17013890/111478676-943ae600-8730-11eb-945f-02ae83c5c647.png">

but if we set the docstring as 
```python
   """
    Returns
    -------
    sentence_embedding_model
        The sentence embedding model instance.
   """
```
we get instead
<img width="357" alt="Screenshot 2021-03-17 at 14 51 53" src="https://user-images.githubusercontent.com/17013890/111478785-aa48a680-8730-11eb-83b3-43a9f3ad4956.png">


@jankrepl so any idea on how to treat the return type in docstrings?

Reply:
My guess is that there are two valid options
1. `variable name : type`
2. `type`

That is why Sphinx did what it did. Maybe there is another syntax that allows for variable names only.